salt/_modules/caasp_hosts.py

```diff
@@ -68,6 +68,19 @@ def __virtual__():
     return "caasp_hosts"
 
 
+def _makedirs(path):
```

Comment:
We should call actually this function make dir singular because it Is creating only 1dir from a var not list